retos/entregas/marioDelrio/RetoCccf.java

```diff
@@ -0,0 +1,179 @@
+import java.util.Scanner;
+
+public class RetoCccf{  
+
+    public static void main(String[]args){
+        
+        Scanner entrada = new Scanner(System.in);
+
+        int tiempoTotal=720;
+        double probabilidadDeLlegada;
+        int numeroCajas=0;
+        int caja1=0;
+        int caja2=0;
+        int caja3=0;
+        int caja4=0;
+        int caja5=0;
+        int cola=0;
+        int cantidadItems1=0;
+        int cantidadItems2=0;
+        int cantidadItems3=0;
+        int cantidadItems4=0;
+        int cantidadItems5=0;
+        int totalItems=0;
+        int personasAtendidas=0;
+        int minutosSincola=0;
+
+        System.out.println("Dime el numero de cajas que quiere que funcionen (1-5)");
+        numeroCajas= entrada.nextInt();
+
+        if(numeroCajas>5){
+            System.out.println("has dicho un numero mayor que 5 no hay ninguna caja abierta reinicia el programa");
+        }   
+

```

Comment:
Para que esto funcione como lo planteas, tendrías que meter el for siguiente dentro del else de este mismo if, tal que:

```
if (numeroCajas>5) {
    System.out.println("has dicho un numero mayor que 5 no hay ninguna caja abierta reinicia el programa");
} else {
    for (int minuto=1; minuto<= tiempoTotal; minuto++ ){
        // TODO LO DEMAS AQUI
    }
}
```